frontend/src/bundles/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -1,66 +1,87 @@
 
 import { DEFAULT_SIGN_UP_PAYLOAD } from './constants/constants.js';
+import { type UserSignUpForm } from './interface.js';
 
 type Properties = {
-    onSubmit: (payload: UserSignUpRequestDto) => void;
+    onSubmit: (payload: UserSignUpForm) => void;
 };
 
 const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
-    const { control, errors, handleSubmit } = useAppForm<UserSignUpRequestDto>({
-        defaultValues: DEFAULT_SIGN_UP_PAYLOAD,
-        validationSchema: userSignUpValidationSchema,
-    });
+    const { control, errors, isValid, handleSubmit, reset } =
+        useAppForm<UserSignUpForm>({
+            defaultValues: DEFAULT_SIGN_UP_PAYLOAD,
+            validationSchema: userSignUpValidationSchema,
+        });
 
     const handleFormSubmit = useCallback(
         (event_: React.BaseSyntheticEvent): void => {
             void handleSubmit(onSubmit)(event_);
+            reset();
         },
-        [handleSubmit, onSubmit],
+        [handleSubmit, onSubmit, reset],
     );
 
     return (
-        <>
-            <h3>Sign Up</h3>
-            <form onSubmit={handleFormSubmit}>
-                <p>
-                    <Input
-                        type="text"
-                        label="Email"
-                        placeholder="Enter your email"
-                        name="email"
-                        control={control}
-                        errors={errors}
-                    />
-                </p>
-                <p>
-                    <Input
-                        type="text"
-                        label="Password"
-                        placeholder="Enter your password"
-                        name="password"
-                        control={control}
-                        errors={errors}
-                    />
-                </p>
+        <div className="bg-lm-black-200 text-lm-white m-auto flex min-h-[988px] max-w-[705px] flex-col items-center justify-center gap-5 rounded-[44px] pb-12 font-sans">
+            <h3 className="text-[30px] font-bold text-slate-50">
+                Hi! Create an account
+            </h3>
+            <p className="text-lm-grey-100 text-xs">or Sign up with Email</p>
+            <form onSubmit={handleFormSubmit} className="mb-10 w-[358px]">
+                <Input
+                    type="text"
+                    label="Email"
+                    placeholder="email@gmail.com"
+                    name="email"
+                    control={control}
+                    errors={errors}
+                />
+
+                <Input
+                    type="password"
+                    label="Password"
+                    placeholder="&bull;"
+                    name="password"
+                    control={control}
+                    errors={errors}
+                />
+
+                <Input
+                    type="password"
+                    label="Confirm Password"
+                    placeholder="&bull;"
```

Comment:
It might be better to remove the need to pass the placeholder prop when type='password' and instead define a constant for the placeholder text near input component.